Up2dateService/Up2dateClient/Client.cs

```diff
@@ -199,10 +180,24 @@ private void OnDeploymentAction(IntPtr artifact, DeploymentInfo info, out Client
                     case InstallPackageStatus.PsScriptInvokeError:
                         additionalMessage = "General Choco Error";
                         break;
+                    case InstallPackageStatus.MsiInstallationError:
+                        additionalMessage = "Msi Installation Error";
+                        break;
+                    case InstallPackageStatus.InstallationPackageIsNotSigned:
+                        additionalMessage = "File not signed. File deleted";
```

Comment:
I think it is better to use term "package" instead of "file", for instance:
"Package is not signed. Deployment rejected"
Same for the next Case

Reply:
Done